modules/xmpp/xmpp.ts

```diff
@@ -481,6 +482,12 @@ export default class XMPP extends Listenable {
             return true;
         }
 
+        if (from === this.pollsComponentAddress) {
```

Comment:
Why is this check different than the other components? Should we be doing things differently overall?

Reply:
yeah ... cause we do not have the type in the json messages 
maybe I can add and the type and that will remove this difference.